src/main/asciidoc/_chapters/developer.adoc

```diff
@@ -1364,17 +1404,13 @@ sudo mount -t tmpfs -o size=2048M tmpfs /ram2G
 You can then use it to run all HBase tests on 2.0 with the command:
```

Comment:
As both commands below are the same, can we just state "You can then use it to run all HBase tests with the command:" and merge the two commands to one?

Reply:
Thats dumb. Thanks for fingering it. I didn't read what was there just blindly changing structure.